package.json

```
@@ -68,6 +68,7 @@
     "update:test-devtools": "bash lighthouse-core/test/chromium-web-tests/test-locally.sh --reset-results",
     "test-devtools": "bash lighthouse-core/test/chromium-web-tests/test-locally.sh",
     "open-devtools": "bash lighthouse-core/scripts/open-devtools.sh",
+    "lighthouse-devtools": "bash lighthouse-core/test/chromium-web-tests/test-page-devtools.sh",
```

🚲 🏠  WDYT about `run-devtools` or `start-devtools` (to match `start` which runs CLI)? everything in here is basically lighthouse :)

```suggestion
    "run-devtools": "bash lighthouse-core/test/chromium-web-tests/test-page-devtools.sh",
```